vortex-serde/src/layouts/read/layouts/inline_dtype.rs

```diff
@@ -81,57 +89,72 @@ impl InlineDTypeLayout {
 
     fn dtype(&self) -> VortexResult<DTypeReadResult> {
         if let Some(dt_bytes) = self.message_cache.get(&[0]) {
-            let msg = root::<message::Message>(&dt_bytes)?
+            let msg = root::<message::Message>(&dt_bytes[FLATBUFFER_SIZE_LENGTH..])?
```

Comment:
nit: this still doesn't read well, maybe this constant should be called `FLATBUFFER_PREFIX_SIZE` because that's what it really is

Reply:
This is the size of the flatbuffer size field, recursive names and so on